src/microbe_stage/MicrobeHUD.cs

```diff
@@ -779,19 +783,25 @@ private void UpdateATP(float delta)
             return;
 
         var atpAmount = 0.0f;
-        var capacity = 4.0f;
 
+        // Update to the player's current HP, unless the player is dead
         if (stage.Player != null)
         {
             var compounds = GetPlayerColonyOrPlayerStorage();
 
-            atpAmount = Mathf.Ceil(compounds.GetCompoundAmount(atp));
-            capacity = compounds.Capacity;
+            atpAmount = compounds.GetCompoundAmount(atp);
+            maxATP = compounds.Capacity;
+        }
+
+        atpBar.MaxValue = maxATP * 10.0f;
+
+        if (maxATP - atpAmount < maxATP / 20.0f)
```

Comment:
Not very obvious what's this for, would be nice to have a small explanation comment here.